example/src/main/java/com/stripe/example/activity/LauncherActivity.java

```diff
@@ -15,8 +15,13 @@
 
 public class LauncherActivity extends AppCompatActivity {
 
+    /*
+     * Change this to your publishable key.
+     *
+     * You can get your key here: https://manage.stripe.com/account/apikeys
```

Comment:
nit: `manage.stripe.com` is a relic from the past – we should really change any `manage.stripe.com` references to `dashboard.stripe.com`.